### ISSUE TITLE
issue since lnd upgrade v0.16.2-beta

### ISSUE THREAD

**Issue by hihouhou** [lightninglabs/lndmon#100]
Hello, I use lndmon since many years but when I updated lnd to v0.16.2-beta I had : 

```
lndmon                | 2023-04-29 20:46:34.428 [INF] LNDMON: Starting Prometheus exporter...
lndmon                | 2023-04-29 20:46:34.429 [INF] HTLC: Starting Htlc Monitor
lndmon                | 2023-04-29 20:46:34.430 [INF] LNDMON: Prometheus active!
lndmon                | Lndmon exiting with error: unknown htlc event type: *routerrpc.HtlcEvent
lndmon                | 2023-04-29 20:46:34.431 [INF] HTLC: Stopping Htlc Monitor
lndmon                | 2023/04/29 20:46:34 Stopping Prometheus Exporter
lndmon                | unknown htlc event type: *routerrpc.HtlcEvent
```
Any ideas ?

Regards,

H.

**Comment by guggero** [comment]
Are you using the latest version of `lndmon`? I think this was fixed in [`v0.2.7`](https://github.com/lightninglabs/lndmon/releases/tag/v0.2.7. We should probably create a proper release to make it more obvious there is a new version out there. Going to do that right now.

**Comment by hihouhou** [comment]
It works, thank you! !!! I confirm for the release, it's better for the visibility ;-)
